src/index.js

```diff
@@ -142,6 +143,12 @@ const ELEMENT_IDS = {
     //
 
     app: "app",
+    viewContact: "view-contact-tab",
+    photo: "photo",
```

Comment:
Again, photo is not great name of anything. What kind of photo is this? Is it dog photo, car photo, contact photo, ...?